jutut/urls.py

```diff
@@ -4,10 +4,12 @@
 import core.urls
 import accounts.urls
 import feedback.urls
+import timeusage.urls
 
 
 urlpatterns = [
     url(r'^', include(feedback.urls)),
+    url(r'^', include(timeusage.urls)),
```

Comment:
This could be added with the prefix, i.e., `url(r'^timeusage/', include(timeusage.urls)),`.